### PR TITLE
push script-runner and trigger-runner to Artifactory

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
I have created the corresponding user and repositories on Artifactory, and tested the `curl` command manually. I'll add the corresponding credentials to Azure once this is approved.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ci/build-unix.yml [hunk 1]
```diff
@@ -92,6 +92,16 @@ steps:
       bazel build //ledger-service/http-json:http-json-binary_deploy.jar
       cp bazel-bin/ledger-service/http-json/http-json-binary_deploy.jar $(Build.StagingDirectory)/$JSON_API
       echo "##vso[task.setvariable variable=json-api;isOutput=true]$JSON_API"
+
+      TRIGGER=trigger-runner-${{parameters.release_tag}}.jar
+      bazel build //triggers/runner:trigger-runner_deploy.jar
+      cp bazel-bin/triggers/runner/trigger-runner_deploy.jar $(Build.StagingDirectory)/$TRIGGER
+      echo "##vso[task.setvariable variable=trigger-runner;isOutput=true]$TRIGGER"
+
+      SCRIPT=script-runner-${{parameters.release_tag}}.jar
```
Comment (cocreature) [inline ci/build-unix.yml:101]: How do you feel about just calling this `daml-script-*.jar`? Matches the CLI command and the name of the feature. (The former also applies to the JSON API but the latter doesn’t so I think this doesn’t invalidate the point)

File: ci/build-unix.yml [hunk 2]
```diff
@@ -92,6 +92,16 @@ steps:
       bazel build //ledger-service/http-json:http-json-binary_deploy.jar
       cp bazel-bin/ledger-service/http-json/http-json-binary_deploy.jar $(Build.StagingDirectory)/$JSON_API
       echo "##vso[task.setvariable variable=json-api;isOutput=true]$JSON_API"
+
+      TRIGGER=trigger-runner-${{parameters.release_tag}}.jar
```
Comment (cocreature) [inline ci/build-unix.yml:96]: Similar question here, how about `daml-trigger-*.jar` or `daml-trigger-runner-*.jar` if we want to reserve the former for the trigger service.

File: ci/build-unix.yml (hunk 1 above)
Comment (garyverhaegen-da) [reply ci/build-unix.yml:101]: No feeling on that, I'll make the change.

File: ci/build-unix.yml (hunk 2 above)
Comment (garyverhaegen-da) [reply ci/build-unix.yml:96]: I have a slight preference for keeping our options open with the service, as that is the long-term one, but ultimately your call. I'll change to `daml-trigger-runner` for now.

Comment (cocreature) [reply ci/build-unix.yml:96]: yeah I also prefer the `daml-trigger-runner` option, thanks!

**Review by cocreature — APPROVED** [review]
Thanks for addressing this so quickly!

**Comment by garyverhaegen-da** [comment]
Added `ARTIFACTORY_USERPASS` to Azure Pipelines.

**Pull request merged**